### ISSUE TITLE
Inconsistent number of datasets

### ISSUE THREAD

**Issue by nebstudio** [bio-ontology-research-group/deepgo2#6]
Hello, I downloaded the data.tar.gz and training-data.tar.gz data you provided on GitHub. When I ran them, I found that the number of proteins in the dataset was inconsistent with that provided in the supplementary file D3 of the paper. For example, why are there 38533, 1901, and 2845 data in the training set, validation set, and test set of mf, respectively? However, the supplementary file D3 of the paper describes why there are 57072, 2964, and 4221 data in the training set, validation set, and test set of mf, respectively?
![3](https://github.com/user-attachments/assets/722b15cd-f099-49ac-9661-a3d7286d8714)
![2](https://github.com/user-attachments/assets/bea42adb-0161-4aad-b809-f69c9d6c0869)

**Comment by Anasus1** [comment]
Any news on this issue?

**Comment by coolmaksat** [comment]
Hi, unfortunately the numbers in the supplement got mixed. But, the data files are ok

**Comment by Anasus1** [comment]
Thank you for the quick reply. So just to confirm if I run these data files with train.py I should get 0.874 for the test auc？

https://www.nature.com/articles/s42256-024-00795-w/tables/1

Just wondering as I got auc above 0.9 for test set, which might be a different issue then.
